src/Config.js

```diff
@@ -280,9 +280,8 @@ export default class Config {
     if (typeof bin === 'string') return bin;
     return toObjectOfStrings(
       bin,
-      `package.json#bin must be an object of strings or a string. See "${
-        this.filePath
-      }"`
+      `package.json#bin must be an object of strings or a string. See "${this
+        .filePath}"`
```

Comment:
Is your editor not using the same version of Prettier as Bolt?

Reply:
Possibly, let me check.